core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -114,6 +120,7 @@ private Optimizely(@Nonnull EventHandler eventHandler,
         this.projectConfigManager = projectConfigManager;
         this.optimizelyConfigManager = optimizelyConfigManager;
         this.notificationCenter = notificationCenter;
+        this.defaultDecideOptions = defaultDecideOptions;
```

Comment:
we should clone it.

Reply:
It's cloned already in builder, so no need for it here.